src/control/cmd/agent/main.go

```diff
@@ -24,15 +24,17 @@
 package main
 
 import (
-	"github.com/daos-stack/daos/src/control/client"
-	"github.com/daos-stack/daos/src/control/common"
-	"github.com/daos-stack/daos/src/control/drpc"
-	"github.com/daos-stack/daos/src/control/log"
-	"github.com/jessevdk/go-flags"
 	"os"
 	"os/signal"
 	"path/filepath"
 	"syscall"
+
+	"github.com/daos-stack/daos/src/control/client"
```

Comment:
Doesn't need to be fixed for this commit, but the convention for imports is:
import (
        // stdlib packages

        // 3rdparty packages

        // internal packages
)

Reply:
Is there a way for the linter to enforce this? I'm not sure what vscode uses for code formatting and linting but it regularly reorganizes these things for me.

Reply:
Well, it's a convention rather than something that could be enforced by a linter. I've been reorganizing imports to follow it whenever I'm editing a file. I should probably just go through and do them all at once. After we've got all of the files following this convention the import churn should quiet down.